### PR TITLE
Nfdiv 446/447/448

### PR DESCRIPTION
**Pull request by velimd**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-446
https://tools.hmcts.net/jira/browse/NFDIV-447
https://tools.hmcts.net/jira/browse/NFDIV-448

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/steps/urls.ts [hunk 1]
```diff
@@ -17,6 +17,9 @@ export const RELATIONSHIP_NOT_BROKEN_URL: PageLink = '/relationship-not-broken';
 export const HELP_WITH_YOUR_FEE_URL: PageLink = '/help-with-your-fee';
 export const HELP_PAYING_HAVE_YOU_APPLIED: PageLink = '/have-you-applied-for-help-with-fees';
 export const HELP_PAYING_NEED_TO_APPLY: PageLink = '/apply-for-help-with-fees';
+export const IN_THE_UK: PageLink = '/in-the-UK';
```
Comment (SebFlippence) [inline src/main/steps/urls.ts:20]: Looks like this will need to be lower case to match the step `src/main/steps/sequence/in-the-uk/content.ts`

Comment (velimd) [reply src/main/steps/urls.ts:20]: surprised it worked locally

Comment (velimd) [reply src/main/steps/urls.ts:20]: might be better to change the folder to captial UK or covert urls to lowercase when finding the file

Comment (velimd) [reply src/main/steps/urls.ts]: i guess the convention is lowercase urls, ill change it to lowercase 👍

**Review by SebFlippence — APPROVED** [review]

**Pull request merged**